erica/erica_legacy/elster_xml/grundsteuer/elster_data_representation.py

```diff
@@ -162,16 +162,16 @@ def __init__(self, input_data: GrundsteuerPayload):
         self.AbsOrt = input_data.eigentuemer.person[0].adresse.ort
         self.Copyright = "(C) 2022 DigitalService4Germany"
 
+        steuernummer_aktenzeichen = generate_electronic_aktenzeichen(input_data.grundstueck.steuernummer,
+                                                                     input_data.grundstueck.adresse.bundesland)
         if input_data.grundstueck.adresse.bundesland in BUNDESLAENDER_WITH_STEUERNUMMER:
             self.OrdNrArt = "S"
-            self.StNr = generate_electronic_steuernummer(input_data.grundstueck.steuernummer,
-                                                         input_data.grundstueck.adresse.bundesland)
+            self.StNr = steuernummer_aktenzeichen
```

Comment:
That is okay because Elster handles that correctly, right?

Reply:
Yes.